keyboards/millet/doksin/keymaps/default/readme.md

```diff
@@ -0,0 +1,4 @@
+# これはDOKСИН(ドクシン)のデフォルトキーマップです
+
+1、と入力されます
+クソほど下らないですね
```

Comment:
Please use English for readmes.

Reply:
Does it support Cyrillic characters?
I'm sorry for causing you trouble.